server/src/main/java/io/druid/server/coordinator/BalancerStrategy.java

```diff
@@ -31,5 +33,10 @@
 
   BalancerSegmentHolder pickSegmentToMove(List<ServerHolder> serverHolders);
 
+  default Iterator<ServerHolder> pickServersToDrop(DataSegment toDropSegment, NavigableSet<ServerHolder> serverHolders)
+  {
+    return serverHolders.descendingIterator();
```

Comment:
Would be nice to include a comment about why this descendingIterator is meaningful. (Like the comment on the old code: `// Use the reverse order to get the holders with least available size first.`)